www/app/Controllers/BaseController.php

```diff
@@ -0,0 +1,104 @@
+<?php 
+namespace App\Controllers;
+
+use Twig\Loader\FilesystemLoader;
+use Twig\Environment;
+use Monolog\Logger;
+use Monolog\Handler\StreamHandler;
+use Monolog\Handler\FirePHPHandler;
+
+/**
+ * Controller basico, contem metodos comuns a todos os outros controllers da
```

Comment:
English in all comments

Reply:
Perfect. I'll be commenting all routines on delivery better